core/templates/dev/head/domain/classroom/classroom-backend-api.service.spec.ts

```diff
@@ -80,27 +67,30 @@ describe('Classroom backend API service', function() {
           responseDictionaries.topic_summary_dicts[1])
```

Comment:
You can move this line where the variable is being assigned to null.

Reply:
At that line the value of `topicSummaryObjectFactory` was null.